hazelcast/src/main/java/com/hazelcast/config/EndpointConfig.java

```diff
@@ -278,6 +281,12 @@ public AltoSocketConfig getAltoSocketConfig() {
         return altoSocketConfig;
     }
 
+    public EndpointConfig setAltoSocketConfig(@Nonnull AltoSocketConfig altoSocketConfig) {
```

Comment:
Missing beta annotation and documentation, same for the getter

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/57f4a326887f98dfdb6329505b31d651c2376e7c